it/test/controllers/UkimsNumberControllerSpec.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import helpers.ItTestBase
+import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
+import play.api.libs.ws.{WSClient, WSRequest}
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+
+class UkimsNumberControllerSpec extends ItTestBase {
+  lazy val client: WSClient = app.injector.instanceOf[WSClient]
+
+  private val url = s"http://localhost:$port$appRouteContext/ukims-number"
```

Comment:
you can use routes.ControllerNameForUkims.onLoad.url to get the url for the endpoint rather than hardcoding it

Reply:
I have make of this now routes.ControllerNameForUkims.onLoad.url

Reply:
After refactoring integration this is no longer needed.